### ISSUE TITLE
Cannot build TypeScript due to use of chalk v5.0.0

### ISSUE THREAD

**Issue by uhyo** [microsoft/TypeScript#46930]
# Bug Report

### TL; DR

[Chalk released 5.0.0 21 hours ago](https://github.com/chalk/chalk/releases/tag/v5.0.0) which is now a Pure ESM module. [TypeScript depends on the latest version of chalk](https://github.com/microsoft/TypeScript/blob/912c01a2a69fdba4a30e508048f8b18517e90f04/package.json#L64), but is incompatible with the ESM version.

We have two options:

- Explicitly depend on chalk 4.x.
- Fix our script to work with chalk 5.0.

### 🔎 Search Terms

chalk ERR_REQUIRE_ESM

### 🕗 Version & Regression Information

main

### ⏯ Playground Link

N/A

### 💻 Code

```sh
npm install
npx gulp watch-tsc`
```

### 🙁 Actual behavior

`npm install`, `npx gulp watch-tsc` or any other attempt to build TypeScript fails with following error:

```
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /path/to/TypeScript/node_modules/chalk/source/index.js
require() of ES modules is not supported.
require() of /path/to/TypeScript/node_modules/chalk/source/index.js from /path/to/personal/TypeScript/scripts/build/utils.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /path/to/TypeScript/node_modules/chalk/package.json.

    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1080:13)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/path/to/TypeScript/scripts/build/utils.js:11:15)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14) {
  code: 'ERR_REQUIRE_ESM'
}
```

### 🙂 Expected behavior

Successfully builds.

**Comment by gmurray81** [comment]
hmm... anyone know a good work around for this until this gets released? have a few builds that didn't have a package-lock that started pulling down the new chalk....

**Comment by sataaa** [comment]
This is still an issue.

**Comment by tiimonn** [comment]
wow at least I know that this is fairly recent I just updated node & npm and went insane

**Comment by mathosmor** [comment]
I have the same problem

**Comment by blendsdk** [comment]
I have the same problem:
`
Error [ERR_REQUIRE_ESM]: require() of ES Module /node_modules/........./chalk/source/index.js from /node_modules/.........../commandline/log.js not supported.
Instead change the require of index.js in /node_modules/........../commandline/log.js to a dynamic import() which is available in all CommonJS modules.
`

**Comment by itssamuelrowe** [comment]
I'm having the same issue. Downgrading to `^4.1.2` fixed the issue for me.

**Comment by Elyx0** [comment]
Hit me today as well.

**Comment by MurhafElmasri** [comment]
me too

**Comment by leonardoviada** [comment]
Happend to me a few minutes ago. Downgrading should not be the solution, but it worked for me.

**Comment by orta** [comment]
I'm going to lock this thread because this is specifically about building the code in this repo, not for any other repo. See the link in the top message if you'd like to learn about how to fix it in your own repos.
